source/help/messaging/attaching-files.rst

```diff
@@ -3,6 +3,9 @@ Attaching Files
 
 --------------
 
+Sharing project plans, new designs, and images can make your messages more 
```

Comment:
maybe emphasize photos, video, voice recordings, files, screenshots, etc.? types we render